.gitignore

```diff
@@ -118,3 +118,5 @@ env/
 
 ### Configuration files ###
 .env
+.vagrant
+/logs.txt
```

Comment:
This works but using a / here suggests you expect a directory rather than a file